scrapy/utils/url.py

```diff
@@ -103,3 +103,16 @@ def guess_scheme(url):
         return any_to_uri(url)
     else:
         return add_http_if_no_scheme(url)
+
+
+def trim_href_attribute(href):
```

Comment:
`strip_html_whitespace` or something like that sounds fine to me, a good idea. In this case we should probably move it somewhere else. 

This function uses whitespace characters defined in html5 standard; just 'strip whitespaces' is `my_string.strip()`.